### PR TITLE
app.js is prat

### PR DESCRIPTION
**Pull request by chaitanyakumar99**
changes has done

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import './App.css';
 function App() {
   return (
     <div className="App">
-      <h2>Hi Hello....</h2>
```
Comment (BijjamVijayReddy) [inline src/App.js:7]: This one is not working

**Review by BijjamVijayReddy — COMMENTED** [review]
Check one more time

File: src/App.js (hunk 1 above)
Comment (chaitanyakumar99) [reply src/App.js:7]: i will change and push

**Comment by chaitanyakumar99** [comment]
Hi

**Pull request merged**